tket/src/PauliGraphRefactor/PauliGraphConverters.cpp

```diff
@@ -306,144 +413,261 @@ pg::PauliGraph circuit_to_pauli_graph3(
   return res;
 }
 
-std::pair<Op_ptr, unit_vector_t> pgop_to_inner_command(
-    const PGOp_ptr& pgop, const qubit_vector_t& qubits) {
+struct IndividualSynth {
+  std::list<ChoiMixTableau::row_tensor_t> rows;
```

Comment:
Yeah, it is just easier to store it as a collection of rows initially so that recursive PGOps (e.g. conditional and controlling structures) can easily just add to the collection. There is an argument that this should motivate adding more features to the tableau structures so we can more clearly just use a tableau here. I'd be happy to do that if you think it would clean things up a bit

Reply:
I think adding some comments to `IndividualSynth` should be enough for now.

Reply:
I've added the comments and changed `rows` to `tableau` as well. The added features on tableau data structures could also be handy in other use cases, so it was no trouble with writing them now